### ISSUE TITLE
KAMI throws a System.AccessViolationException when attempting to run with Ratchet: Deadlocked (PCSX2)

### ISSUE THREAD

**Issue by OnceACodeMonkey** [isJuhn/KAMI#39]
KAMI v0.0.2
PCSX2 v1.7.0 dev 1177
.NET version 5.0.11
Ratchet: Deadlocked SCUS_974.65

KAMI kept crashing whenever I tried to use it with Ratchet: Deadlocked (and R&C 3 as well, for that matter), so I built it on my own system and ran it with Visual Studio to see what was causing the crash. PineICP.cs throws the System.AccessViolationException from the Version function when KAMI tries to display the game info whenever I boot Ratchet: Deadlocked.

**Comment by isJuhn** [comment]
Yes this is an issue with pineIPC which I think occurs when you call a function like getVersion when there is no connection. We can keep this open to track it here anyway since it impacts us. Related to #15
